docs/system-requirements.md

```diff
@@ -0,0 +1,44 @@
+# System requirements
+
+## Server
+Additional to normal Nextcloud requirements the following constraints apply to use Nextcloud Talk:
```

Comment:
We should talk about increased server load due to signalling. For example, a few weeks ago I failed using Talk on a decent shared webhosting package ([all-inkl.com Premium](https://all-inkl.com/webhosting/premium/)) with about 25 active users. Impossible to use - and I'm talking about just Chat here.

```suggestion
Nextcloud Talk will significantly increase load on your server, depending on the number of concurrent active conversations - no matter whether they use Audio, Video or just Chat. This is due to required signalling. By default Talk will use its integrated signalling server, which works just fine for small and medium installations. If you're running Talk on a dedicated or decent virtual server, you shouldn't experience any problems. However, if you're running Talk on a shared webhoster, you might indeed experience issues. Please also note that many shared webhosters particularly prohibit running chat server software like Talk; please make sure to check your providers's Terms & Conditions. In this case you might be required to upgrade to either a dedicated server or use Talk's [High-performance Back-end (HPB)](https://www.spreed.eu/contact-nextcloud-talk-high-performance-backend/) for signalling.

Additional to normal Nextcloud requirements the following constraints apply to use Nextcloud Talk:
```

Reply:
I modified it a bit to be less "aggressive"